set.mm

```diff
@@ -498438,6 +498438,60 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       TZBWPTZRZYCBWSTZRSAYDYFYCAWPWSBYBWFWGAYEYDYCRSYDYCWHHWIWJWK $.
   $}
 
+  ${
+    $d x G $.  $d x V $.
+    lfuhgr.1 $e |- V = ( Vtx ` G ) $.
+    lfuhgr.2 $e |- I = ( iEdg ` G ) $.
+    $( A hypergraph is loop-free if and only if every edge connects at least
```

Comment:
On the Wikipedia page https://en.wikipedia.org/wiki/Hypergraph your notion of 'loop-free' is close (but not exactly the same) to a 'simple' hypergraph.  Does it make sense to use consistently the Wikipedia nomenclature? Also a loop can be misunderstood as a closed cycle within the hypergraph.

Reply:
I'm just reusing the definition from [section 16.2.3](https://us.metamath.org/mpeuni/mmtheorems266.html#mm26530s) of set.mm.